src/config.ts

```diff
@@ -0,0 +1,4 @@
+export interface Config {
+  realm: string;
+  users: string[];
```

Comment:
This type is not correct. Users is an object with `id`, `username`, `email` and `name` properties.